solidity/random-beacon/contracts/libraries/Groups.sol

```diff
@@ -14,17 +14,28 @@ library Groups {
 
     struct Group {
         bytes groupPubKey;
-        uint256 activationTimestamp;
+        uint256 activationBlockNumber;
         uint32[] members;
+        // When selected group does not create a relay entry on-time it should
+        // be marked as terminated.
+        bool terminated;
     }
 
     struct Data {
         // Mapping of keccak256 hashes of group public keys to groups details.
         mapping(bytes32 => Group) groupsData;
         // Holds keccak256 hashes of group public keys in the order of registration.
         bytes32[] groupsRegistry;
+        // Group ids that were active but failed creating a relay entry.
+        uint64[] activeTerminatedGroups;
```

Comment:
If the group was terminated when it was active but now it is expired, it is not included here, right? If yes, I think it's worth stating explicitly in the comment above.

Reply:
Sure, it is already described [here](https://github.com/keep-network/keep-core/blob/groups-expiration/solidity/random-beacon/contracts/libraries/Groups.sol#L294) but also added a comment above b72dafa